projects/plugins/jetpack/modules/stats.php

```diff
@@ -461,6 +554,21 @@ function stats_reports_page( $main_chart_only = false ) {
 			?>
 			</h2>
 		</div>
+		<div class="wrap">
+			<div class="stats-odyssey-notice stats-odyssey-notice--content__highlighted">
+				<div class="stats-odyssey-notice--content">
+					<h2 class="stats-odyssey-notice--content-header"><?php esc_html_e( 'Old Jetpack Stats experience', 'jetpack' ); ?></h2>
+					<p class="stats-odyssey-notice--content-text"><?php esc_html_e( 'The old Jetpack Stats has been deprecated and will be removed soon.', 'jetpack' ); ?></p>
```

Comment:
Maybe "The old Jetpack Stats has been deprecated and will be removed soon. Please click the button to enable the new experience."